openbb_terminal/stocks/fundamental_analysis/av_view.py

```diff
@@ -216,7 +216,13 @@ def display_earnings(
 
 @log_start_end(log=logger)
 @check_api_key(["API_KEY_ALPHAVANTAGE"])
-def display_fraud(ticker: str, export: str = "", help_text: bool = False):
+def display_fraud(
+    ticker: str,
+    export: str = "",
+    help_text: bool = False,
+    color: bool = True,
```

Comment:
let's have color True and not being optional

Reply:
Done!